addons/calendar/tests/test_calendar.py

```diff
@@ -162,13 +147,11 @@ def test_event_allday(self):
             'name': 'All Day',
             'start': "2018-10-16 00:00:00",
             'start_date': "2018-10-16",
-            'start_datetime': False,
             'stop': "2018-10-18 00:00:00",
             'stop_date': "2018-10-18",
-            'stop_datetime': False,
             'allday': True,
         })
-
+        event.invalidate_cache()
```

Comment:
Remove this.

Reply:
You sure ? The test is crashing without that line.